### PR TITLE
update animals table

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Vet Clinic database 🐶 
> Query and Update animals table

In this PR I : 
* Add column species in animals table
* Add more data to animals table
* Set species value to unspecified then roll back in a transaction
Screenshot from 2022-09-12 15-19-41
* Update species value to `digimon` for animals `name` ending by `mon` and `pokemon` for the rest
Screenshot from 2022-09-12 15-27-56
* Delete all data from animals inside a transaction then roll back
Screenshot from 2022-09-12 15-31-49
Screenshot from 2022-09-12 15-32-39
* Inside a transaction 
  * Delete all animals born after Jan 1st 2022
  * Create a savepoint
  * Update all animals' weight to be their weight multiplied by -1
  * Rollback to the savepoint
  * Update all animals' weights that are negative to be their weight multiplied by -1.
 
Screenshot from 2022-09-12 15-34-50

* Write queries to : 
  * Display the total number of animals
  * Display the number of animals that never tried to escape
  * Display average weight of animals
  * Verify which escapes the most between the neutered and not neutered animals
  
Screenshot from 2022-09-12 15-41-23

  * Display the minimum and the maximum weight of each species in animals
  
Screenshot from 2022-09-12 15-42-06

  * Display the average number of escapes per species in animals born between 1990 and 2000
 
Screenshot from 2022-09-12 15-42-42

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals VALUES(1,'Agumon','2020-02-03',0,'1',10.23);
 INSERT INTO animals VALUES(2,'Gabumon','2018-11-15',2,'1',8);
 INSERT INTO animals VALUES(3,'Pikachu','2021-01-07',1,'0',8);
 INSERT INTO animals VALUES(4,'Agumon','2017-05-12',5,'1',11);
+INSERT INTO animals VALUES(5,'Charmander','2020-02-08',0,'0',11);
+INSERT INTO animals VALUES(6,'Plantmon','2021-11-15',2,'1',5.7);
+INSERT INTO animals VALUES(7,'Squirtle','1993-04-02',3,'0',12.13);
+INSERT INTO animals VALUES(8,'Angemon','2005-06-12',1,'1',45);
```
Comment (cynthiainga) [inline data.sql]: - [ ]  Please kindly add the `-` to your weights is the negative numbers, not positive number the values are `-11.0, -5.7, -12.13, -45.0` 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @ChrispaixK, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 

## Required Changes ♻️

- [ ]  Good use of the database readme template but it could be better if you customize it and put the necessary information in your README file 🙃 🙃
- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (ChrispaixK) [reply data.sql]: Thank you @cynthiainga

**Review by brytebee — APPROVED** [review]
Hi, @ChrispaixK👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**